app/Bucketlist/bucket.py

```diff
@@ -0,0 +1,380 @@
+        if data:
+            try:
+                new_bucket_name = data['new_bucket_name']
+            except Exception as e:
+                abort(401, message='Missing new data.')
+
+            try:
+                bucket = get_bucketlist(
+                    get_user_id(request.headers.get('Authorization')),
+                    bucket_id
+                )
+            except Exception as e:
+                return jsonify(
+                    {
+                        'message':'Bucket doesn\'t exist',
+                        'status_code':404
+                    })
+            
+            try:
+                edited_bucket_list = edit_bucketlist(
+                        get_user_id(request.headers.get('Authorization')),
+                        bucket_id,
+                        new_bucket_name,
+                        datetime.datetime.now()
+                    )
+                    
+                return jsonify({
+                    'message':'Bucketlist edited successful',
+                    'status_code':200
+                    })
+
+            except Exception as e:
+                abort(401, message='Invalid or Missing access token ')
+        else:
+            abort(401, message='No bucketlist data passed.')
+        
+    def delete(self, bucket_id):
+
+        try:
+            get_user_id(request.headers.get('Authorization'))
+        except Exception as e:
+            abort(401, message='Access forbidden, Login to continue.')
+
+        try:
+            bucket = get_bucketlist(
+                get_user_id(request.headers.get('Authorization')),
+                bucket_id
+            )
+
+        except Exception as e:
+            return jsonify(
+                {
+                    'message':'Bucket doesn\'t exist',
+                    'status_code':404
+                })
+
+        try:
+            bucket_deleted = delete_bucketlist(
+                get_user_id(request.headers.get('Authorization')),
+                bucket_id)
+
+            return jsonify({
+                'message':'Bucket deleted successfull.',
+                'status_code':200
+                })
+        except Exception as e:
+            return jsonify(
+                {
+                    'message':'Bucket doesn\'t exist',
+                    'status_code':404
+                })  
+    
+class BucketListItem(Resource):
+    
+    def get(self, bucket_id):
+        return jsonify({"message":"Create a new bucket item by sending"
+                        " POST request to /bucketlists/<int:bucket_id>/items."})
+
+    def post(self, bucket_id):
+
+        try:
+            get_user_id(request.headers.get('Authorization'))
+        except Exception as e:
+            abort(401, message='Access forbidden, Login to continue.')
+
+        data = json.loads(request.get_data(as_text=True))
+        if data:
+            try:
+                new_item_name = data['item_name']
```

Comment:
Best to always use `get` to access keys. It'll prevent a `KeyError` exception from being thrown when a `key` doesn't exist